commerce-frontend-js/dev/public/index.html

```diff
@@ -25,6 +25,8 @@
 	<ul>
 		<li>
 			<a href="/example.html">Example</a>
+			<a href="/dataset-display.html">Dataset Display</a>
+			<a href="/sidepanel.html">Side Panel</a>
```

Comment:
This should really be `side_panel.html` rather than `sidepanel.html` since we treat this as 2 words elsewhere